README.md

```diff
@@ -212,13 +214,17 @@ App password permissions required are:
     
     Documentation on how to generate an app password can be found [here](https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html).
 
-* `AZURE_DEVOPS_SERVICES` uses personal access tokens which will eventually expire. In the event of token expiring, the integration will need to be deleted and recreated (either manually or via the plugin).
+* `AZURE_DEVOPS_SERVICES` and `AZURE_DEVOPS_SERVER` use personal access tokens which will eventually expire. In the event of token expiring, the integration will need to be deleted and recreated (either manually or via the plugin).
 
     See Microsoft documentation here for details on creating a Personal Access Token. Please ensure that the token has at minimum the scope Code (read and write) for the organization you wish to access
 
 * A list of `AZURE_DEVOPS_SERVICES` organizations that you currently have access to can be found [here](https://aex.dev.azure.com/me). Also listed are the projects for each organization.
 
-    Enter just the organization, without ‘dev.azure.com’ at the beginning or ‘.visualstudio.com’ at the end. For example, if your organization is ‘dev.azure.com/example-user’ or 'example-user.visualstudio.com’, enter just 'example-user’. 
+    Enter just the organization, without ‘dev.azure.com’ at the beginning or ‘.visualstudio.com’ at the end. For example, if your organization is ‘dev.azure.com/example-user’ or 'example-user.visualstudio.com’, enter just 'example-user’.
+
+* `AZURE_DEVOPS_SERVER` is a local instance of Azure DevOps services, thus corresponding url must be provided e.g. http://locahost:3434
+
+* `AZURE_DEVOPS_SERVER` contains ProjectCollection property which must be included into configurations with its default value - `DefaultCollection`
```

Comment:
You can remove the above 2 lines, anyone familiar with TFS/DevOps Server would be able be knowledgeable enough to understand these properties in conjunction with the above table of properties.